WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -7,6 +7,13 @@ struct CartView: View {
     @Environment(\.floatingControlAreaSize) var floatingControlAreaSize: CGSize
     @Environment(\.dynamicTypeSize) var dynamicTypeSize
 
+    @State private var offSetPosition: CGFloat = 0.0
+    private var coordinateSpace: CoordinateSpace = .global
```

Comment:
To properly check the scrolling position, we need to use a `ScrollView` coordinate space, not a global coordinate space. This way we can check the precise position of elements (`VStack`) within that `CoordinateSpace`:

```suggestion
    private var coordinateSpace: CoordinateSpace = .named("ScrollView")
```

And set coordinateSpace modifier for ScrollView:

````

.coordinateSpace(name: "ScrollView")
```

Reply:
🎉 https://github.com/woocommerce/woocommerce-ios/pull/13725/commits/98f6cb6a533d82783bc056f90b083fb152a11550